Fallback treatment for swaps (#5768)
rotki/rotki: docs/changelog.rst

* :bug:`-` Ignoring forked assets ETC, BCH and BSV for accounting should now also remove any pre-fork references of them and completely omit them from the PnL report.
* :bug:`-` Users with kraken accounts with old data that were never purged and repulled will no longer have missing events.
* :bug:`-` PnL report will now correctly show progress bar percentage if user has connected but non-syncing exchanges.

> * :bug:`5717` Swaps will now be processed in accounting correctly no matter the counterparty set.

Reviewer: Ah and one more comment on changelog. Remember user-facing.

```suggestion
* :bug:`5717` Swaps will now be processed in accounting correctly even if set manually.
```
